sql/src/main/java/io/crate/metadata/Scalar.java

```diff
@@ -79,6 +80,8 @@
     public Symbol normalizeSymbol(Function symbol, TransactionContext transactionContext) {
         try {
             return evaluateIfLiterals(this, symbol);
+        } catch (UnsupportedFeatureException e) {
```

Comment:
pls add the note in the code, no one is looking at the commit message afterwards ... you mean you changed if for debugging purposes? wondering why this is actually not a change in contract. probably evaluation of literals must not throw exceptions at all?